docs/source/docs/cli.md

```diff
@@ -286,7 +286,7 @@ The saved files include:
 
   Example usage:
 
-  - `--image input_DDF.nii.gz`
+  - `--image input_DDF.nii.gz`##
```

Comment:
what is this ##?

Reply:
@s-sd don't forget this XD thx!

Reply:
@mathpluscode, I don't think I modified the cli.md file so I'm not sure about this

Reply:
I changed it ;)